### PR TITLE
String updates

### PR DESCRIPTION
**Pull request by marcellamaki**
Much string cleanup - particularly for the setup wizard strings, but other tidying here and there. Random strings have just been added to commonCore

### CODE REVIEW

File: kolibri/plugins/setup_wizard/assets/src/views/SelectSuperAdminAccountForm.vue [hunk 1]
```diff
@@ -202,21 +202,21 @@
...
         context: 'Explanation of what the super admin account is used for on device.',
       },
       chooseAdminPrompt: {
-        message: "Choose an admin from '{facility}' or create a new super admin.",
+        message: "Choose an admin from '{facility}' learning facility or create a new super admin.",
         context: 'Prompt that goes above a select input of options for admins',
       },
       enterPasswordPrompt: {
-        message: "Enter the password for '{username}'",
+        message: "Enter the password for '{username}' in {facility_name} learning facility",
```
Comment (radinamatic) [inline kolibri/plugins/setup_wizard/assets/src/views/SelectSuperAdminAccountForm.vue]: Shouldn't the `{facility_name}` also be enclosed by single quotes like `'{username}'`?

Comment (rtibbles) [reply kolibri/plugins/setup_wizard/assets/src/views/SelectSuperAdminAccountForm.vue]: Yes.

> Always wrap user-generated text in 'single quotes'

https://design-system.learningequality.org/writing/#punctuation

**Comment by rtibbles** [comment]
One failing test, so you don't have to go hunting for it:
```
FAIL kolibri/plugins/device/assets/src/views/__test__/PinAuthenticationModal.spec.js
  ● PinAuthenticationModal › submitting a PIN › on submit, shows a validation message when PIN code is empty

    expect(received).toEqual(expected) // deep equality

    Expected: "Please enter a 4-digit number"
    Received: "Enter four numbers to set as your new PIN"

      63 |       const textbox = getElements(wrapper).pinTextbox();
      64 |       expect(textbox.props().invalid).toEqual(invalid);
    > 65 |       expect(textbox.props().invalidText).toEqual(errorText);
         |                                           ^
      66 |     }
      67 |
      68 |     it('if user has not interacted with the form, then no validation messages appear', () => {

      at toEqual (kolibri/plugins/device/assets/src/views/__test__/PinAuthenticationModal.spec.js:65:43)
      at assertTextbox (kolibri/plugins/device/assets/src/views/__test__/PinAuthenticationModal.spec.js:115:11)
```

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 2]
```diff
@@ -1027,7 +1027,7 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     context: 'Section header for the list of resources that are related to the current resource',
   },
   doNotShowAgain: {
-    message: "Don't show this again",
+    message: "Don't show this message again",
```
Comment (rtibbles) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js:1035]: The message here now seems to mismatch with the context - the context is about resources, not messages.

File: kolibri/core/assets/src/mixins/taskStrings.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ const taskStrings = createTranslator('TaskStrings', {
     context: 'Displays the user that started a task',
   },
   taskLODFinishedByLabel: {
-    message: '‘{fullname}’ from ‘{facilityname}’ successfully loaded to this device',
+    message: 'Account ‘{fullname}’ from ‘{facilityname}’ successfully loaded to this device',
```
Comment (rtibbles) [inline kolibri/core/assets/src/mixins/taskStrings.js]: We have inconsistent single quotes - I imagine we want to update all `‘` to `'`

File: kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue [hunk 4]
```diff
@@ -1002,7 +1002,7 @@
       },
       allowLearnersDownloadDescription: {
         message:
-          "Allow users to explore resources they don't have and mark it for Kolibri to automatically download when it's available in their network.",
+          'Allow users to explore resources they don’t have and mark it for Kolibri to automatically download when it’s available in their network.',
```
Comment (rtibbles) [inline kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue]: See my note above about consistent punctuation. I think we probably don't want to be using the `‘`. Either leave with the double quotes, or escape the `\'`.

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 5]
```diff
@@ -1325,11 +1366,138 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
...
+    message:
+      'A learning facility represents the location where you are installing Kolibri, such as a school, training center, or your home.',
+    context: '',
+  },
+  joinLearningFacility: {
+    message: 'Join learning facility',
+    context: '',
+  },
+  joinedSuccessfully: {
+    message: 'You have successfully joined the {facility} learning facility',
```
Comment (radinamatic) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js]: Shouldn't `{facility}` be enclosed in single quotes?

File: kolibri/plugins/device/assets/src/views/PinAuthenticationModal.vue [hunk 6]
```diff
@@ -80,7 +80,7 @@
         context: 'Error message displayed when an incorrect PIN is input',
       },
       invalidPin: {
-        message: 'Please enter a 4-digit number',
+        message: 'Enter four numbers to set as your new PIN',
```
Comment (rtibbles) [inline kolibri/plugins/device/assets/src/views/PinAuthenticationModal.vue:83]: Flagging that this change is the reason for the failing test, so this string needs to be updated in the test too.

File: kolibri/plugins/setup_wizard/assets/src/views/ImportIndividualUserForm.vue [hunk 7]
```diff
@@ -360,6 +360,10 @@
         message:
           '’{full_name} ({username})’ is an admin on ‘{device}’. This device is limited to features for learners only. Features for coaches and admins will not be available.',
```
Comment (rtibbles) [inline kolibri/plugins/setup_wizard/assets/src/views/ImportIndividualUserForm.vue]: `‘` again.

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 8]
```diff
@@ -1325,11 +1366,138 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
...
+  },
+  enterPassword: {
+    message: 'Enter password',
+    context: '',
+  },
+
+  // post setup
+  preLoadedContentWelcomeText: {
+    message:
+      'Welcome to the {facility} learning facility. Your class materials can be found on the home page.',
```
Comment (radinamatic) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js]: Same as above: shouldn't {facility} be enclosed in single quotes?

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/FullOrLearnOnlyDeviceForm.vue [hunk 9]
```diff
@@ -60,7 +60,7 @@
       },
       fullDeviceDescription: {
         message:
-          'This device will be a fully featured Kolibri server used by admins, coaches and learners',
+          'This device will be a fully-featured Kolibri server used by admins, coaches, and learners.',
```
Comment (rtibbles) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/FullOrLearnOnlyDeviceForm.vue:63]: I approve.

File: kolibri/plugins/user_profile/assets/src/views/ChangeFacility/ConfirmChangeFacility.vue [hunk 10]
```diff
@@ -99,7 +99,7 @@
       },
       changeFacilityInfoLine2: {
         message:
-          'Your user account type will change from ‘{role} to ‘learner’. You will need someone with admin permissions to change your account type to ‘{role}’ again.',
+          'Your user account type will change from ‘{role}’ to ‘learner’ and you will no longer be able to manage resources on this device. You will need someone with admin permissions in ‘{facility}’ to change your account type back to ‘{role}’.',
```
Comment (rtibbles) [inline kolibri/plugins/user_profile/assets/src/views/ChangeFacility/ConfirmChangeFacility.vue]: More `‘`

**Review by rtibbles — CHANGES_REQUESTED** [review]
One string that no longer seems to match its context, and some inconsistent punctuation.

File: kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue (hunk 4 above)
Comment (radinamatic) [inline kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue]: After discussing with @rtibbles, we propose to reword as:

`'Allow users to mark resources for Kolibri to download when available in their network.'`

**Review by radinamatic — APPROVED** [review]
One string to slim down, and a couple of user-input facility names are missing the single quote enclosures, but otherwise seems good to go! 👏🏽 💯 🎉

**Review by rtibbles — APPROVED** [review]
All comments addressed.

**Pull request merged**